internal/manifests/collector/parser/processor/processor_k8sattributes.go

```diff
@@ -50,15 +50,24 @@ func (o *K8sAttributesParser) ParserName() string {
 }
 
 func (o *K8sAttributesParser) GetRBACRules() []rbacv1.PolicyRule {
-	var prs []rbacv1.PolicyRule
-
-	// This one needs to be added always
-	policy := rbacv1.PolicyRule{
-		APIGroups: []string{""},
-		Resources: []string{"pods", "namespaces"},
-		Verbs:     []string{"get", "watch", "list"},
+	// These policies need to be added always
+	var prs []rbacv1.PolicyRule = []rbacv1.PolicyRule{
+		{
+			APIGroups: []string{""},
+			Resources: []string{"pods", "namespaces"},
+			Verbs:     []string{"get", "watch", "list"},
+		},
+		{
+			APIGroups: []string{"apps"},
+			Resources: []string{"replicasets"},
+			Verbs:     []string{"get", "watch", "list"},
+		},
+		{
+			APIGroups: []string{"extensions"},
+			Resources: []string{"replicasets"},
+			Verbs:     []string{"get", "watch", "list"},
+		},
```

Comment:
Why do we need this? I don't think any of our supported K8s versions ship this version.

Reply:
When it is not added, the processor doesn't work: #2823, https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/32247

Reply:
I mean `extensions/replicaset` specifically. That API group was removed in K8s 1.16: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#replicaset-v116

Reply:
You're right. Let me test if it works without them 🤔